playbooks/ci-ceph-swift-rhel/templates/heat_stack.yml

```diff
@@ -93,97 +93,8 @@ parameters:
 resources:
 
```

Comment:
can you make sure your spacing is consistent across your changes to the templates?